python/arcticdb/version_store/_normalization.py

```diff
@@ -1016,7 +1023,7 @@ def _nested_msgpack_unpackb(buff, raw=False):
 
     def _custom_pack(self, obj):
         if isinstance(obj, pd.Timestamp):
-            tz = obj.tz.zone if obj.tz is not None else None
+            tz = str(obj.tz) if obj.tz is not None else None
```

Comment:
It seems dangerous to rely on `str` for this, as most libraries don't give API guarantees for it.

How about:

```
if obj.tz is None:
  tz = None
elif obj.tz.zone is None:
   tz = obj.tzname()
else:
   tz = obj.tz.zone
```
?

I also think we have the same problem elsewhere in this file, eg:

```
def _to_tz_timestamp(dt):
    # type: (datetime.datetime)->(int, str)
    ts = pd.Timestamp(
        year=dt.year,
        month=dt.month,
        day=dt.day,
        hour=dt.hour,
        minute=dt.minute,
        second=dt.second,
        microsecond=dt.microsecond,
    ).value
    tz = dt.tzinfo.zone if dt.tzinfo is not None else None
    return ts, tz
```